samples/v1.0/Elements/Container.SelectAction.json

```diff
@@ -1,50 +1 @@
-{
-	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-	"type": "AdaptiveCard",
-	"version": "1.0",
-	"body": [
-		{
-			"type": "Container",
-			"items": [
-				{
-					"type": "TextBlock",
-					"text": "Cool link"
-				}
-			],
-			"selectAction": {
-				"type": "Action.OpenUrl",
-				"url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
-			}
-		},
-		{
-			"type": "Container",
-			"style": "emphasis",
-			"items": [
-				{
-					"type": "TextBlock",
-					"text": "Emphasis link"
-				}
-			],
-			"selectAction": {
-				"type": "Action.OpenUrl",
-				"url": "https://msn.com"
-			}
-		},
-		{
-			"type": "Container",
-			"items": [
-				{
-					"type": "TextBlock",
-					"text": "Submit action"
-				}
-			],
-			"selectAction": {
-				"type": "Action.Submit",
-				"title": "Submit action",
-				"data": {
-					"info": "My submit action data"
-				}
-			}
-		}
-	]
-}
+{
	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.0",
	"body": [
		{
			"type": "Container",
			"items": [
				{
					"type": "TextBlock",
					"text": "Cool link"
				}
			],
			"selectAction": {
				"type": "Action.OpenUrl",
                         "tooltip":"Go to a url",
				"url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
			}
		},
		{
			"type": "Container",
			"style": "emphasis",
			"items": [
				{
					"type": "TextBlock",
					"text": "Emphasis link"
				}
			],
			"selectAction": {
				"type": "Action.OpenUrl",
                         "tooltip":"Go to a different url",
				"url": "https://msn.com"
			}
		},
		{
			"type": "Container",
			"items": [
				{
					"type": "TextBlock",
					"text": "Submit action"
				}
			],
			"selectAction": {
				"type": "Action.Submit",
				"tooltip": "Submit action",
				"data": {
					"info": "My submit action data"
				}
			}
		}
	]
}

```

Comment:
nit: Different line endings are observed in this card compared to other cards when diff using GitHub from webbrowser.

Reply:
Fixed!